src/Npgsql/Internal/Converters/Primitive/DoubleConverter.cs

```diff
@@ -1,16 +1,43 @@
+using System.Numerics;
+
 // ReSharper disable once CheckNamespace
 namespace Npgsql.Internal.Converters;
 
-sealed class DoubleConverter : PgBufferedConverter<double>
+sealed class DoubleConverter<T> : PgBufferedConverter<T>
```

Comment:
It's back!

Reply:
Yeah, I figured if we move this out to a plugin, might as well be consistent with it :/